ts/Stock/StockToolsBindings.ts

```diff
@@ -2047,41 +2047,50 @@ var stockToolsBindings: Record<string, Highcharts.NavigationBindingsOptionsObjec
             button: HTMLDOMElement
         ): void {
             var chart = this.chart,
-                series = chart.series[0],
-                options = series.options,
-                lastVisiblePrice = (
-                    options.lastVisiblePrice &&
-                    options.lastVisiblePrice.enabled
-                ),
-                lastPrice = options.lastPrice && options.lastPrice.enabled,
+                series = chart.series,
+                priceIndicatorEnabled = chart.stockTools && chart.stockTools.priceIndicatorEnabled,
```

Comment:
Question: Is it safe to store this as a property? Can something change the button state, so we should check it on demand (`isPriceIndicatorEnabled(..)`)? For example on series add/remove?

Reply:
I still have a small problem with storing `priceIndicatorEnabled` variable in `stockTools`. It is a bit bloating the properties, that are related to the tools in general (HTML references, options, chart references etc.). A property specific to just one button.

Can we go for now with the function call in both places? Upon `chart.render` event and here, when clicking on a button. So below would be changed from:

```js
if (priceIndicatorEnabled) {
  ...
}
```
to:
```js
if (chart.stockTools.isPriceIndicatorEnabled(series)) {
 ...
}
```

Or even better, store `isPriceIndicatorEnabled` in `bindingsUtils`, and use it:
```js
if (bindingsUtils.isPriceIndicatorEnabled(series)) {
 ...
}
```

What do you think?